logic/dapp.js

```diff
@@ -43,12 +43,12 @@ __private.unconfirmedAscii = {};
  * @todo Add description for the params
  */
 class DApp {
-	constructor(db, logger, schema, network) {
+	constructor(logger, schema, network, storage) {
```

Comment:
Why not just replace the `db` with `storage`. Why you have to change the order of params here?

Reply:
Db was removed after whole implementation. Will re-order and fix tests.

Reply:
Done.